fastscanner/adapters/cmd/run_eodhd_provider.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import time
+from pathlib import Path
+
+from fastscanner.adapters.fundamental.eodhd import EODHDFundamentalStore
+from fastscanner.pkg.logging import load_logging_config
+
+import sys
+sys.path.insert(0, str(Path(__file__).parent.parent.parent))
```

Comment:
Please don't add this to the scripts. You need to run the scripts from the root folder with
```
python fastscanner/adapters/cmd/run_eodhd_provider.py
```
I just noticed you also added it to the other script. Please delete it from there too